src/test/regress/expected/multi_remove_node_reference_table.out

```diff
@@ -141,7 +141,7 @@ WHERE colocationid IN
      WHERE logicalrelid = 'remove_node_reference_table'::regclass);
```

Comment:
Shall we add one test to `multi_metadata_sync ` so that we won't get into such weird cases again? We already know the exact point where to add the test, so should be easy.

```SQL
-- THIS IS THE SECOND TIME WE CREATE mx_ref
 CREATE TABLE mx_ref (col_1 int, col_2 text);
 SELECT create_reference_table('mx_ref');

-- make sure that adding/removing nodes wouldn't end up
-- with multiple colocation entries for reference tables
SELECT count(*) FROM pg_dist_colocation WHERE distributioncolumntype = 0;
 count 
-------
     1
(1 row)
```